responders/src/API/EMBC.Responders.API/Controllers/SuppliersController.cs

```diff
@@ -85,11 +85,11 @@ public async Task<ActionResult<Supplier>> GetSupplierById(string supplierId)
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         public async Task<ActionResult<SupplierResult>> CreateSupplier([FromBody] Supplier supplier)
         {
-            if (supplier.Team == null)
+            if (supplier.PrimaryTeams == null)
             {
-                supplier.Team = new SupplierTeamDetails();
+                supplier.PrimaryTeams = new[] { new SupplierTeamDetails() };
             }
-            supplier.Team.Id = teamId;
+            supplier.PrimaryTeams.FirstOrDefault().Id = teamId;
```

Comment:
initialize correctly; also, when creating a supplier it should always be the team in the user context, no matter what the frontend passes